consensus/types/src/chain_spec.rs

```diff
@@ -229,6 +229,7 @@ pub struct ChainSpec {
     pub max_blocks_by_root_request: usize,
     pub max_blocks_by_root_request_deneb: usize,
     pub max_blobs_by_root_request: usize,
+    pub max_data_columns_by_range_request: usize,
 
```

Comment:
We should leave these DAS changes out of this PR. We want to focus only on `Electra` stuff since this will be merged into `unstable`

Reply:
removed, sorry about that